Avoid adding new packages that aren't necessary for the manual page.
--- a/doc/en/conf.py
+++ b/doc/en/conf.py
@@ -27,16 +27,11 @@
 
 root_doc = "index"
 extensions = [
-    "pygments_pytest",
     "sphinx.ext.autodoc",
     "sphinx.ext.autosummary",
     "sphinx.ext.intersphinx",
     "sphinx.ext.todo",
     "sphinx.ext.viewcode",
-    "sphinx_removed_in",
-    "sphinxcontrib_trio",
-    "sphinxcontrib.towncrier.ext",  # provides `towncrier-draft-entries` directive
-    "sphinx_issues",  # implements `:issue:`, `:pr:` and other GH-related roles
 ]
 
 # Building PDF docs on readthedocs requires inkscape for svg to pdf
